docs/output.md

```diff
@@ -404,11 +405,32 @@ Besides the reference files or output files created by BUSCO, the following summ
 
 </details>
 
+### Bin Refinement
```

Comment:
Since this might be the input for the above mentioned `QUAST` and `BUSCO` steps, should we move this accordingly?